Makefile

```diff
@@ -124,6 +135,7 @@ endif
 
 generate: generate-code kubefedctl
 	./scripts/sync-up-helm-chart.sh
+	./scripts/check-generated-bindata-content.sh
```

Comment:
The script being called here should be updating the bindata content, not checking it.  `check-make-generate-output` in `pre-comment.sh` calls `make generate` and then verifies that the working tree has not changed.